haystack/components/evaluators/answer_exact_match.py

```diff
@@ -6,11 +6,13 @@
 @component
 class AnswerExactMatchEvaluator:
     """
-    Evaluator that checks if predicted answers exactly match ground truth answers.
+    An answer exact match evaluator class.
+
+    The evaluator that checks if the predicted answers matches any of the ground truth answers exactly.
+    The result is a number from 0.0 to 1.0, it represents the proportion any predicted answer
```

Comment:
```suggestion
    The result is a number from 0.0 to 1.0, it represents the proportion of predicted answers
```

This has to be either "...the proportion of predicted answers..." or "...represents any predicted answer that matched..."